packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/clinicalDataGrid.1.doc.jsx

```diff
@@ -3,7 +3,7 @@ import DocTemplate from 'terra-doc-template';
 import ReadMe from '../../../../docs/README.md';
 import { name } from '../../../../package.json';
 // Component Source
-import DataGridSrc from '!raw-loader!../../../../src/DataGrid';
+import DataGridPropsTable from '!terra-props-table-loader!../../../../src/DataGrid';
```

Comment:
@mjhenkes @bjankord   Should this be 
```diff
import DataGridPropsTable from '!terra-props-table-loader!../../../../src/DataGrid';
import DataGridPropsTable from ../../../../src/DataGrid?'terra-props-table-loader;
```
And should this be loading from `lib`?

Reply:
not yet, the resource query will be applied with mdx dev site and no, we want this to load from source.